sycl/source/detail/compiler.hpp

```diff
@@ -78,13 +78,58 @@
 
 #define __SYCL_PROGRAM_METADATA_TAG_NEED_FINALIZATION "Requires finalization"
 
+// New entry type after
```

Comment:
Do we want to specify here that this is a replica of the EntryTy data structure in llvm/include/llvm/Frontend/Offloading/Utility.h?

Thanks